Core/Core/Planner/PlannerViewController.swift

```diff
@@ -19,47 +19,130 @@
+    var studentID: String?
 
-    var calendar: CalendarViewController!
-    var plannerList: PlannerListViewController!
-    var studentID: String!
-
-    public static func create(studentID: String) -> PlannerViewController {
-        let vc = PlannerViewController()
-        vc.studentID = studentID
-        return vc
+    public static func create(studentID: String? = nil) -> PlannerViewController {
+        let controller = PlannerViewController()
+        controller.studentID = studentID
+        return controller
     }
 
     public override func viewDidLoad() {
         super.viewDidLoad()
 
-        calendar = CalendarViewController.create(studentID: studentID)
-        plannerList = PlannerListViewController.create(studentID: studentID)
-        calendar.delegate = plannerList
+        embed(listPageController, in: view)
+        listPageController.dataSource = self
+        listPageController.delegate = self
+        listPageController.setViewControllers([
+            PlannerListViewController.create(
+                start: Clock.now.startOfDay(),
+                end: Clock.now.startOfDay().addDays(1),
+                delegate: self
+            ),
+        ], direction: .forward, animated: false)
+        for view in listPageController.view.subviews {
+            if let scroll = view as? UIScrollView {
+                scroll.canCancelContentTouches = true
+            }
+        }
 
-        embed(calendar, in: view, constraintHandler: { child, container in
+        embed(calendar, in: view) { child, container in
             child.view.pinToLeftAndRightOfSuperview()
             child.view.topAnchor.constraint(equalTo: container.topAnchor).isActive = true
-            child.view.bottomAnchor.constraint(equalTo: container.bottomAnchor).isActive = true
-        })
+        }
 
-        embed(plannerList, in: view, constraintHandler: { [weak self] child, container in
-            guard let ss = self else { return }
-            child.view.pinToLeftAndRightOfSuperview()
-            NSLayoutConstraint.activate([
-                child.view.topAnchor.constraint(equalTo: ss.calendar.daysContainer.bottomAnchor),
-                child.view.bottomAnchor.constraint(equalTo: container.bottomAnchor),
-            ])
-        })
+        let divider = DividerView()
+        divider.tintColor = .named(.borderMedium)
+        divider.isOpaque = false
+        view.addSubview(divider)
+        divider.pinToLeftAndRightOfSuperview()
+        divider.heightAnchor.constraint(equalToConstant: 1).isActive = true
+        divider.topAnchor.constraint(equalTo: calendar.view.bottomAnchor).isActive = true
+
+        calendar.view.layoutIfNeeded()
+        list.tableView.scrollIndicatorInsets.top = calendar.minHeight
+        list.tableView.contentInset.top = calendar.minHeight
+    }
+
+    func getPlannables(from: Date, to: Date) -> GetPlannables {
+        return GetPlannables(userID: studentID, startDate: from, endDate: to)
```

Comment:
this top-level controller owns the creation of the use case, so that filters, selected user, context, etc are consistent across list & calendar.